Makefile

```diff
@@ -49,9 +49,10 @@ cflags-filter.sh:
 		export CC="${CC}";					\
 		cd ${LIBCPERCIVA_DIR}/POSIX;				\
 		command -p sh posix-cflags-filter.sh "$$PATH";		\
-	else								\
-		:;							\
 	fi > $@
+	if [ ! -s $@ ]; then						\
+		printf "#define CFLAGS_OK 1\n";				\
```

Comment:
I know that you were quite happy that `cpusupport-config.h` could act as both a header file and a shell script, so I've followed that here.  But I'm wondering if this message should be `# CFLAGS_DEFAULT are ok` instead, i.e. not keeping this file as a C99 header.  I don't feel strongly either way.

Reply:
I cared about `cpusupport-config.h` being both a header file and a shell script because we needed both and it was important that they were in sync -- making it just a single file solved the "keep in sync" problem.

Since we're only using `cflags-filter.sh` as a shell script, there's no need to worry about that.  I'd probably spell the message `# Compiler understands normal flags; nothing to filter out` though.